packages/graphql-proxy-server/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "events-helsinki-graphql-proxy-server",
+  "description": "Events Helsinki GraphQL proxy.",
+  "version": "0.0.1",
```

Comment:
Could this version be set for example to 2.0.0?

Reply:
done